cmd/neofs-cli/modules/object/util.go

```diff
@@ -483,3 +480,72 @@ func collectObjectRelatives(ctx context.Context, cmd *cobra.Command, cli *client
 
 	return chain
 }
+
+type payloadWriter struct {
+	payload []byte
+}
+
+func (pw *payloadWriter) Write(p []byte) (n int, err error) {
+	pw.payload = append(pw.payload, p...)
+	return len(p), nil
+}
+
+func getChildrenFromV2Link(ctx context.Context, cmd *cobra.Command, cli *client.Client, key *ecdsa.PrivateKey, linkID oid.ID) ([]oid.ID, bool) {
+	var addrObj oid.Address
+	var prmGet internal.GetObjectPrm
+	pw := &payloadWriter{}
+
+	addrObj.SetObject(linkID)
+	prmGet.SetClient(cli)
+	prmGet.SetPrivateKey(*key)
+	prmGet.SetAddress(addrObj)
+	prmGet.SetRawFlag(false)
```

Comment:
no need to set default explicitly

same below

Reply:
this was copy-pasted. dropped